api_tests/nodes/views/test_node_list.py

```diff
@@ -3392,7 +3403,7 @@ def test_skip_uneditable_bulk_update(
         edited = res.json['data']
         skipped = res.json['errors']
         assert_equals(
-            [edited[0]['id'], edited[1]['id']],
+            [edited[1]['id'], edited[0]['id']],
```

Comment:
This was fixed. It was switched because I was importing the wrong assert_equals. `assert_items_equal` was renamed to `assert_equals` during the py3 upgrade. I did not know this until recently.